internal/examples/proxy/http2tchannel/README.md

```diff
@@ -0,0 +1,43 @@
+This is an example that illustrates an HTTP to TChannel transport translation
+proxy for a single service in a Docker composition.
+The "caller" services sends HTTP requests, the "proxy" service receives HTTP
+requests and forwards them over TChannel to the "service".
+
+To run this example, make the Linux binaries for each of the services
+(`*/main.go`) and then bring up the services in Docker.
+
+```
+make
```

Comment:
You should update the readme to show that they need to have Go install and have this repo checked out in the correct place in their `$GOPATH`.